src/main/java/ch/njol/skript/expressions/ExprElement.java

```diff
@@ -83,33 +95,37 @@ private enum ElementType {
 	private Expression<? extends T> expr;
 	private	@Nullable Expression<Integer> startIndex, endIndex;
 	private ElementType type;
+	private boolean queue;
 
 	@Override
 	@SuppressWarnings("unchecked")
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
 		ElementType[] types = PATTERNS.getInfo(matchedPattern);
-		expr = LiteralUtils.defendExpression(exprs[exprs.length - 1]);
+		this.queue = matchedPattern > 4;
```

Comment:
I'm not sure this is true. These patterns conflict (due to the earlier ones accepting any type). You probably need to put them first.

Reply:
No, this is the only way that works. If you put the queue syntax first then all the elements/collections tests fail.
<img width="405" alt="image" src="https://github.com/user-attachments/assets/c89c425e-b126-4c43-b096-4c8609e07d17">